git_greppers/gitlab_handler.sh

```
@@ -154,4 +154,121 @@ find_gitlab_mr_info() {
+  if ! command -v glab &> /dev/null; then
+    debug 1 "GitLab CLI not available, skipping recent MR fetching"
+    # Create an empty array with the given name
+    eval "$output_array_name=()"
+    return 0
+  fi
+  
+  # Change to the repo directory
+  cd "$repo_path" || return 1
+  
+  # Get date string for the search query
+  local search_date=$(date -v-${date_range}d +%Y-%m-%d)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)